### PR TITLE
Add post about Cargo team changes

### PR DESCRIPTION
**Pull request by ehuss**
This adds an internals post about some changes in the Cargo team.

cc @rust-lang/cargo

### CODE REVIEW

File: posts/inside-rust/2022-03-25-cargo-team-changes.md
Comment (calebcartwright) [inline posts/inside-rust/2022-03-25-cargo-team-changes.md:28]: :rofl:

**Comment by Mark-Simulacrum** [comment]
@ehuss I want to check in that we have sign-off here from @alexcrichton and @epage as they're prominently mentioned in the blog post. Did that happen at a Cargo meeting or elsewhere, either on direct text or on general idea? Want to avoid surprises.

I also think it's worth considering whether there's some slightly more positive tone we might be able to adopt towards the end -- obviously, presuming it's not misleading. Maybe we can say something about looking forward to the new team members helping to expand the review capacity as they ramp up on the full breadth of Cargo?

**Review by epage — APPROVED** [review]
I'm good with my part

**Review by alexcrichton — APPROVED** [review]
Indeed looks good to me!

**Comment by joshtriplett** [comment]
@Mark-Simulacrum I think the team is comfortable with this not coming across as positive. This is *intended* to be concerning. I *am* hopeful that @epage and @weihanglo will help us on this front, but I don't expect that to put us in a position where we have the capacity for new features anytime soon.

**Comment by Mark-Simulacrum** [comment]
Ok! In part I bring it up as it might be a good opportunity to try to identify how we can change that (e.g., what can folks who want to help do). It's quite possible the answer is nothing yet, though; I'm not sure whether it make sense to make that more explicit than just asking for patience. (The reverse effect of people contacting in a storm asking what they can do may also be good to mitigate with slightly more explicit language).

In any case, happy to merge this when ready - I think the date will need to get bumped. Please ping on Zulip so it doesn't get lost in email.

File: posts/inside-rust/2022-03-25-cargo-team-changes.md
Comment (nikomatsakis) [inline posts/inside-rust/2022-03-25-cargo-team-changes.md:50]:
```suggestion
to be done very carefully. It is easy for changes to break some
```

**Review by nikomatsakis — DISMISSED** [review]
Minor wording nit.

**Review by Mark-Simulacrum — APPROVED** [review]

**Pull request merged**
